recipes/icu/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: icu
+  version: 56.1
+
+source:
+  fn:  icu4c-56_1-src.tgz
+  url: http://download.icu-project.org/files/icu4c/56.1/icu4c-56_1-src.tgz
```

Comment:
Could we refactor the version information using jinja? Guessing we need to variables for this to keep it clean.